docs/site/src/layouts/MainLayout.astro

```diff
@@ -125,5 +125,6 @@ const githubEditUrl = `${CONFIG.GITHUB_EDIT_URL}${currentPage === '/' ? '/index'
 				<RightSidebar {content} {githubEditUrl} />
 			</aside>
 		</main>
+		<script src="//cdn.jsdelivr.net/npm/@builder.io/persist-attribution@latest/dist/persist-attribution.min.js" async></script>
```

Comment:
```suggestion
		<script src="//cdn.jsdelivr.net/npm/@builder.io/persist-attribution@latest/dist/persist-attribution.min.js" async defer id="persist-attribution-init" data-send-page-view-events="true"></script>
```
The new version of this script now contains code to track page view events, however this code is disabled by default, so to enable that we need the `id` and the `data-send-page-view-events` attributes.